scifight/views.py

```diff
@@ -62,5 +62,5 @@ def problems(request, tournament_slug):
     return HttpResponse('This is problems page')
 
 
-def problem(request, tournament_slug, problem_id):
-    return HttpResponse('This is problem  #%s page' % problem_id)
\ No newline at end of file
+def problem(request, tournament_slug, problem_num):
+    return HttpResponse('This is problem  #%s page' % problem_num)
```

Comment:
Always notice those «No newline at end of file» marks, they're here for reason ([an obscure and historic reason][1]).

[1]: http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline